examples/c/ksyscall.bpf.c

```diff
@@ -0,0 +1,27 @@
+#include "vmlinux.h"
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+
+#define TASK_COMM_LEN 16
+
+SEC("ksyscall/kill")
+int BPF_KSYSCALL(entry_probe, pid_t tpid, int sig) 
```

Comment:
what's tpid? manpage says it's always process ID and is called `pid`. If you'd like to trace thread kill syscalls, there is `tgkill()` syscall. Perhaps let's update the example to trace both and distinguish between them. Seems like a bit more fun example?